tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -859,7 +859,7 @@ and you left the Cuda or cuDNN version empty, try specifying them explicitly.
 ### Protobuf library related issues
 
 TensorFlow pip package depends on protobuf pip package version
-3.0.0b2. Protobuf's pip package downloaded from [PyPI](https://pypi.python.org)
+3.0.0. Protobuf's pip package downloaded from [PyPI](https://pypi.python.org)
```

Comment:
See my other comment about updating:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/pip_package/setup.py

Seems like it's available on pypi to update:
https://pypi.python.org/pypi/protobuf/3.1.0